manifests/init.pp

```diff
@@ -90,6 +91,7 @@
   $jvm_xmx                                                          = '1024m',
   $jvm_permgen                                                      = '256m',
   $jvm_optional                                                     = '-XX:-HeapDumpOnOutOfMemoryError',
+  $jvm_extra                                                        = '-XX:+PrintGCDateStamps',
```

Comment:
We can enforce the datatype here as well:
```suggestion
  String $jvm_extra                                                 = '-XX:+PrintGCDateStamps',
```

Reply:
I've updated this.